.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+---
+
+**Which package/packages do you use?**
+
+- [ ] `@stream-io/video-react-sdk`
+- [ ] `@stream-io/video-react-native-sdk`
+- [ ] `@stream-io/video-client`
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Version**
+
+- SDK version
+
+**Env:**
+
+- OS: [e.g. iOS, linux...]
+- Browser [e.g. chrome, safari]
+- Emulator [e.g. android emulator, real phone...]
```

Comment:
`Device:`

We should also preferably add device here.

Reply:
```suggestion
- Device: [e.g. android emulator, real phone...]
```
Yeah, I also think we should replace `Emulator` with `Device` here.

Reply:
Makes sense, I've also separated env info based on packages, please check: https://github.com/GetStream/stream-video-js/pull/846/commits/e175296540204ddd87045b563c39f59c1d2b31e1

Reply:
Perfect! 😄